src/auth-service/utils/request-access.js

```diff
@@ -67,11 +72,18 @@ const requestAccess = {
         ).list({
           filter,
         });
-        if (responseFromListUsers.message === "successfully listed the users") {
+        if (
+          responseFromListUsers.message ===
+          "successfully retrieved the user details"
+        ) {
+          logger.error(
+            `candidate ${email} already exists as a User in the System`
+          );
           callback({
-            success: true,
-            message: "candidate already exists as a user",
-            status: httpStatus.OK,
+            success: false,
```

Comment:
During request access, send appropriate response in case User already exists as a User in the system. @geed254 @wabinyai @Codebmk @BenjaminSsempala